tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -182,6 +182,7 @@ module.exports = {
 
       await this.useXpath()
         .click(this.getFileLinkSelectorByFileName(folder, 'folder'))
+        .waitForAjaxCallsToStartAndFinish()
         .useCss()
```

Comment:
Does it need to wait after the folder has been clicked/opened?

Reply:
No, that's not needed. Reloading the page before clicking the folder fixes the issue. That must me leftover. I'll remove it